samples/core/Component from Airflow operator/Component from Airflow operator.ipynb

```diff
@@ -0,0 +1,110 @@
+{
```

Comment:
Should we add licensing terms to the beginning of the file? 

```
# Copyright 2019 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#      http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.

```

Reply:
For notebook sample it's a bit of hard I guess?
And putting it as a markdown cell would be a bit of awkward, WDYT?

Reply:
Usually we don't do that for the notebooks.
JSON format does not support comments >_<